packages/orchestrator-ui-components/src/graphqlQueries/tasksQuery.ts

```diff
@@ -0,0 +1,49 @@
+import { parse } from 'graphql';
+import { gql } from 'graphql-request';
+
+import type { TypedDocumentNode } from '@graphql-typed-document-node/core';
+
+import {
+    GraphqlQueryVariables,
+    TaskDefinition,
+    TaskDefinitionsResult,
+} from '@/types';
+
+export const GET_TASKS_GRAPHQL_QUERY: TypedDocumentNode<
+    TaskDefinitionsResult,
+    GraphqlQueryVariables<TaskDefinition>
+> = parse(gql`
+    query MetadataTasks(
+        $first: Int!
+        $after: Int!
+        $sortBy: [GraphqlSort!]
+        $query: String
+    ) {
+        workflows(
+            first: $first
+            after: $after
+            sortBy: $sortBy
+            query: $query
+            filterBy: { field: "target", value: "SYSTEM" }
```

Comment:
Are you sure about the results of this query? I seem to remember there are processes that don't have the target SYSTEM but do have isTask=true. You miss those in this query